src/overlay/dead_npcs_box.py

```diff
@@ -0,0 +1,82 @@
+import pygame
+
+from src.npc.dead_npcs_registry import DeadNpcsRegistry
+from src.settings import OVERLAY_POSITIONS
+from src.support import get_translated_string, import_font, import_image
+
+
+class DeadNpcsBox:
+    def __init__(self, dead_npcs_registry: DeadNpcsRegistry):
+        # setup
+        self.display_surface = pygame.display.get_surface()
+        self.dead_npcs_registry = dead_npcs_registry
+        self.img_size = (15, 20)
+        self.image = pygame.transform.scale(
+            import_image("images/objects/cross.png"), self.img_size
+        )
+
+        # dimensions
+        self.left = 20
+        self.top = 50
+
+        width, height = 700, 55
+        self.font = import_font(20, "font/LycheeSoda.ttf")
+
+        self.rect = pygame.Rect(self.left, self.top, width, height)
+
+        self.rect.topleft = OVERLAY_POSITIONS["dead_npcs_box"]
+
+    def display(self):
+        black = "Black"
+        background_color = "Red"
+        foreground_color = black
```

Comment:
I'd suggest just using constants instead of redefining these variables at every call